src/main/content/_assets/js/collections.js

```diff
@@ -1,82 +1,155 @@
+        .then(function (response) {
+            return response.json()
+        })
+        .then(handleInstancesRequests)
+        .catch(error => console.error("Error syncing collections", error))
+}
+
+function updateCollectionView(collectionJSON) {
+    if (typeof collectionJSON === "undefined") {
         return;
     }
-  
+
     let collections = collectionJSON["kabanero collections"];
 
     collections.forEach(coll => {
         $("#collection-table-body").append(createCollRow(coll));
     });
 
-    // hide loader table and show this one
-    $(".table-loader").hide();
-    $("#collection-table").show();
-
-    function createCollRow(coll){
+    function createCollRow(coll) {
         let row = $("<tr>");
         let name = $("<td>").text(coll.name);
         let version = $("<td>").text(coll.version);
         let status = $("<td>").text(coll.status);
-        return row.append([name, version, status]);
+        let deactivateCollection = createDeactivateCollectionButton(coll);
+        return row.append([name, version, status, deactivateCollection]);
     }
+
+    function createDeactivateCollectionButton(coll) {
+        let iconStatus = coll.status === "active" ? "icon-active" : "icon-disabled";
+        let deactivateCollection = $("<td>").addClass("deactivate-collection-td");
+        let div = $("<div>").addClass(`deactivate-collection-icon ${iconStatus}`).attr("collection-name", coll.name).attr("data-modal-target", "#deactivate-collection-modal-" + iconStatus);
+        let svg =`<svg focusable="false" preserveAspectRatio="xMidYMid meet" xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 32 32" aria-hidden="true" style="will-change: transform;"><path d="M16,4A12,12,0,1,1,4,16,12,12,0,0,1,16,4m0-2A14,14,0,1,0,30,16,14,14,0,0,0,16,2Z"></path><path d="M10 15H22V17H10z"></path><title>Deactivate ${coll.name} collection</title></svg>`
+
+        div.append(svg);
+        deactivateCollection.append(div);
+
+        return deactivateCollection;
+    }
+    showTable();
 }
 
-function setCLIVersion(cliVersion){
+function setCLIVersion(cliVersion) {
     if(typeof cliVersion === "undefined"){
         return;
     }
-
     let version = cliVersion["image"].split(":")[1];
-
     $("#cli-version").append(version);
-    $("#table-header-cli-version").show();
+}
+
+function hideAndEmptyTable(){
+    $(".table-loader").show();
+    $("#table-footer-cli-version").hide();
+    $("#collections-sync-button").hide();
+    $("#collection-table").hide();
+    $("#collection-table-body").empty();
+    $("#cli-version").empty();
+    $("#instance-accordion").empty();
+}
+
+function showTable(){
+    // hide loader table and show table with data
+    $(".table-loader").hide();
+    $("#collection-table").show();
+    $("#collections-sync-button").show();
+    $("#table-footer-cli-version").show();
+    $("#cli-version").show();
+    $("#collections-sync-button").show();
```

Comment:
I think `showTable` is not needed. On load we already had the carbon loading table, then its replaced with the real table with data.

When we want to sync the table all we should need to do is:
1. hide the real table `$("#collection-table").hide();`
1. show the loading table `$(".table-loader").show();`
1. delete all the table rows from the real table `$("#collection-table tr").remove();`  (or something like that)
1. sync refills the table by calling `handleCollectionsRequests()`
1. hide loading table show the the loading table